### PR TITLE
feat: quick filters

### PR DESCRIPTION
**Pull request by daibhin**
## Problem

We ran an experiment on quick filters and think that this design performs as well for folks and should make the most common use cases easier

## Changes

- Remove experiment feature flag
- Add scrolling to filters
- Add prompt to add person properties
- Fix scrolling in settings pages

Screenshot 2023-08-17 at 14 02 24

## How did you test this code?

By 👁️

### CODE REVIEW

File: frontend/src/scenes/session-recordings/filters/SimpleSessionRecordingsFilters.tsx [hunk 1]
```diff
@@ -82,41 +85,50 @@ export const SimpleSessionRecordingsFilters = ({
                         })
                     }}
                 />
+                {displayNameProperties.length === 0 && (
+                    <Link to={urls.projectSettings('person-display-name')}>
+                        <LemonButton type="tertiary" size="small" className="whitespace-nowrap" icon={<IconPlus />}>
+                            Add person properties
```
Comment (pauldambra) [inline frontend/src/scenes/session-recordings/filters/SimpleSessionRecordingsFilters.tsx:91]: 💯

File: frontend/src/lib/hooks/useAnchor.ts [hunk 2]
```diff
@@ -11,11 +11,8 @@ export function useAnchor(hash: string): void {
             }
 
             element.classList.add('highlighted')
-            const elementPosition = element.getBoundingClientRect().top + window.pageYOffset
-            window.scrollTo({
-                top: elementPosition - 50 - 32, // compensate for header & top margin of pages
-                behavior: 'smooth',
-            })
+
```
Comment (pauldambra) [inline frontend/src/lib/hooks/useAnchor.ts:14]: noticed this only works for me the second time I click the link

(or rather not the first time after a refresh)

File: frontend/src/scenes/session-recordings/filters/SimpleSessionRecordingsFilters.tsx [hunk 3]
```diff
@@ -47,7 +50,7 @@ export const SimpleSessionRecordingsFilters = ({
 
     return (
         <div className="space-y-2">
-            <div className="flex space-x-1">
+            <div className="flex space-x-1 px-3 overflow-x-scroll pb-3">
```
Comment (pauldambra) [inline frontend/src/scenes/session-recordings/filters/SimpleSessionRecordingsFilters.tsx:53]: should this have `overflow-x-scroll` or should it wrap?

Comment (daibhin) [reply frontend/src/scenes/session-recordings/filters/SimpleSessionRecordingsFilters.tsx:53]: Great shout! Wrapping is much nicer

File: frontend/src/lib/hooks/useAnchor.ts (hunk 2 above)
Comment (daibhin) [reply frontend/src/lib/hooks/useAnchor.ts:14]: Fixed with a `window.requestAnimationFrame`

**Pull request merged**